kafka_consumer/datadog_checks/kafka_consumer/client/kafka_python_client.py

```diff
@@ -29,14 +29,6 @@ def token(self):
 
 
 class KafkaPythonClient(KafkaClient):
-    def __init__(self, config, tls_context, log) -> None:
```

Comment:
How come we wouldn't need this part anymore?

Reply:
because I moved it to the superclass https://github.com/DataDog/integrations-core/pull/14022/files#diff-414e25ef3b0e9d3114346cba1b8500545ef67e2bf3598b76b1c6d9b6f9dbcd7eR8-R14

Reply:
Ah gotcha, missed that!